### PR TITLE
fix: add Download directory

### PR DESCRIPTION
**Pull request by Lumograul**

### CODE REVIEW

**Review by 1martin1 — COMMENTED** [review]
remove comments

File: InstanceManager/Services/DownloadService.cs [hunk 1]
```diff
@@ -8,8 +7,9 @@ public class DownloadService
     public void DownloadFile(string address, string fileName)
     {
         WebClient myWebClient = new WebClient();
-        Console.WriteLine("Downloading File \"{0}\" from \"{1}\" .......\n", fileName, address);
+        var pathName = Path.Combine("Download", fileName);
+        //Console.WriteLine("Downloading File \"{0}\" from \"{1}\" .......\n", fileName, address);
```
Comment (1martin1) [inline InstanceManager/Services/DownloadService.cs:11]: remove this and next comment

Comment (Lumograul) [reply InstanceManager/Services/DownloadService.cs:11]: they will be needed for logging

**Review by 1martin1 — APPROVED** [review]
looks valid

**Pull request merged**